File: TA_P3/BackEnd/LuminaBeauty/LuminaBeauty-Model/src/main/java/pe/edu/pucp/luminaBeauty/Model/Cliente.java
package pe.edu.pucp.luminaBeauty.Model;

public class Cliente extends Usuario {
    // El idUsuario lo hereda de Usuario
    private int puntosFidelidad;
    private String nivelCliente; // En SQL es un ENUM, en Java lo manejamos como String

    public Cliente() {
        super(); // Llama al constructor de Usuario
    }

    // Getters y Setters específicos de Cliente
    public int getPuntosFidelidad() {
        return puntosFidelidad;
    }

    public void setPuntosFidelidad(int puntosFidelidad) {
        this.puntosFidelidad = puntosFidelidad;
    }

    public String getNivelCliente() {
        return nivelCliente;
    }

    public void setNivelCliente(String nivelCliente) {
        this.nivelCliente = nivelCliente;
    }
}

File: lumina_beauty_final_version/lumina_beauty_frontend/LuminaBeauty/Servicios/Modelo/Cliente.cs
using System.Text.Json.Serialization;

namespace LuminaBeauty.Servicios.Modelo
{
    public class Cliente : Usuario
    {
        [JsonPropertyName("puntos_fidelidad")]
        public int PuntosFidelidad { get; set; }

        [JsonPropertyName("nivel_cliente")]
        public string NivelCliente { get; set; } = "BRONCE";


    }
}
